docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
```

Comment:
I think i meant more, we would need to split the `upgrade` mode up based on whether or not this was a deployment where only fluentd is used. And was wondering if you'd imagine us still using a single `upgrade` mode, or also have one specific for a solo fluentd deployment?

We currently try to detect the current stack version based on what features are installed. If its a solo fluentd we wouldn't have any other components and the deployer would think its at a version before curator was introduced since it would be absent... 

I guess we could add a label/annotation on the fluentd daemonset to indicate to ourselves that it is solo and if it is absent then we know its a prior version..